test/timecop_test.rb

```diff
@@ -397,6 +397,14 @@ def test_travel_time_with_block_returns_the_value_of_the_block
     assert_equal expected, actual
   end
 
+  def test_local_time_with_same_components_is_equal_the_current_time
+    Timecop.travel(2008, 10, 10, 10, 10, 10) do
```

Comment:
You probably want to use freeze to stop time, otherwise time will change as the test is executing causing `current_time` and `same_time` to be different.